pages/find-locations/index.md

```diff
@@ -1,8 +0,0 @@
----
-title: Find VA Locations
-layout: page-react.html
-description: Find a VA medical center, clinic, hospital, national cemetery, or VA benefit office near you. You can search by city, state, zip code, or service. You'll get wait times and directions.
-keywords: VA location by zip code, VA clinic, VA near me
-entryname: facilities
----
-
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-0eb2db0ef00932d1a41c421b543096a3R5. Breadcrumbs are controlled by the React app.